frontend/src/scenes/session-recordings/sessionRecordingLogic.ts

```diff
@@ -165,6 +195,29 @@ export const sessionRecordingLogic = kea<sessionRecordingLogicType>({
                 sessionPlayerData?.snapshots.length > 1 && // more than one snapshot needed to init rrweb Replayer
                 !!sessionPlayerData?.snapshots?.find((s: eventWithTime) => s.type === 2), // there's a full snapshot in the data that was loaded
         ],
+        eventsApiParams: [
+            (selectors) => [selectors.sessionPlayerData],
+            (sessionPlayerData) => {
+                if (
+                    !sessionPlayerData?.person?.id ||
+                    !sessionPlayerData?.session_recording?.start_time ||
+                    !sessionPlayerData?.session_recording?.recording_duration
+                ) {
+                    return null
+                }
+
+                const buffer_ms = clamp(sessionPlayerData.session_recording.recording_duration / 4, 0, 30000) // +- before and after start and end of a recording to query for.
+                return {
+                    person_id: sessionPlayerData.person.id,
+                    after: dayjs.utc(sessionPlayerData.session_recording.start_time).subtract(buffer_ms, 'ms').format(),
+                    before: dayjs
+                        .utc(sessionPlayerData.session_recording.start_time)
+                        .add(buffer_ms + sessionPlayerData.session_recording.recording_duration, 'ms')
```

Comment:
Duration from the server is in seconds not ms, so I think this is causing weird behavior.

Reply:
good catch! looks like s and ms are being used together.